lib/src/main/java/graphql/nadel/Nadel.kt

```diff
@@ -38,15 +41,12 @@ import java.util.concurrent.atomic.AtomicReference
 import java.util.function.Function
 
 class Nadel private constructor(
-    private val engine: NadelExecutionEngine,
+    private val engine: NextgenEngine,
```

Comment:
Removed the engine abstraction. Nadel itself creates the engine now. All constructor params that were tied to `NextgenEngine` are now in `Nadel.Builder`.